app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
- [ ] Hey there! 🌟 According to the project requirements, it's important to include email confirmation during user registration. 😊 Unfortunately, in the existing setup, this feature is missing. To address this, please make sure to add the :confirmable module to Devise. 🔄 Additionally, in the Devise migration file, uncomment the confirmable column and then run the migration again. This step ensures that new users must confirm their email before being able to log in to the app. 📧👍

![Screenshot from 2024-01-16 17-39-19](https://github.com/Surafels/Blog-app/assets/112261237/a348af7e-8343-4a14-9fbf-b0c0f9dd3d7b)

```suggestion
  	devise :database_authenticatable, :registerable,
				:recoverable, :rememberable, :validatable, :confirmable
```